packages/libraries/core/package.json

```diff
@@ -42,6 +42,7 @@
     "@fluentui-react-native/pressable": "0.9.12",
     "@fluentui-react-native/radio-group": "0.15.15",
     "@fluentui-react-native/separator": "0.12.4",
+    "@fluentui-react-native/switch": "0.6.8",
```

Comment:
We've been erring towards not bloating the core library because of bundle size concerns.

I think we should move towards not having the core bundle and just have people import components separately. Maybe have a min-bundle that includes the very basic things, like Text and Button. But it's not useful for people to have all the components.